eox_tagging/api/v1/viewset.py

```diff
@@ -36,6 +37,32 @@ def get_queryset(self):
 
         return queryset
 
+    def create(self, request, *args, **kwargs):
+        """"Hijack the create method and use a wrapper function to perform the
+        audit process. The original parameters of create are not very useful in
+        raw form, this way we pass more useful information to our wrapper
+        function to be audited
+        """
+
+        @audit_method(action="eox_tagging-api-v1-viewset:create")
```

Comment:
if we create a new viewset in the same folder and file there will not be a way to  know which viewset is